pytorch_lightning/profiler/pytorch.py

```diff
@@ -402,14 +391,11 @@ def start(self, action_name: str) -> None:
 
         if (
             self.profiler is not None
-            and (
-                any(action_name.endswith(func) for func in self._record_functions)
```

Comment:
do we still need self._record_functions and RECORD_FUNCTIONS/RECORD_FUNCTION_PREFIX?

Reply:
Removed. Thanks

Reply:
RECORD_FUNCTION_PREFIX seems still no used?